main/options/options.cc

```diff
@@ -1199,6 +1201,24 @@ void readOptions(Options &opts,
             }
         }
 
+        if (raw.count("packager-layer") && !opts.stripePackages) {
+            // Default values are available at raw[...], but don't show up in raw.count(...),
+            // so we can use this as a way to check if the user passed in the flag
+            logger->error("--packager-layer can only be specified in --stripe-packages mode");
+            throw EarlyReturnWithCode(1);
+        }
```

Comment:
Doesn't this imply that the default value is never actually used?

If you pass `--stripe-packages` and don't pass `--package-layer`, doesn't this code raise an error?

Reply:
Ah I guess not, I see what your comment says now.